clientinitializer.go

```diff
@@ -195,36 +202,53 @@ func initializeClients() {
 
 	params.Path = v1Addressable
 	params.Url = metaAddr + params.Path
-	svc.ac = metadata.NewAddressableClient(params, types.Endpoint{})
+	svc.ac = metadata.NewAddressableClient(params, consulEndpoint)
 
 	params.Path = v1Device
 	params.Url = metaAddr + params.Path
-	svc.dc = metadata.NewDeviceClient(params, types.Endpoint{})
+	svc.dc = metadata.NewDeviceClient(params, consulEndpoint)
 
 	params.Path = v1DevService
 	params.Url = metaAddr + params.Path
-	svc.sc = metadata.NewDeviceServiceClient(params, types.Endpoint{})
+	svc.sc = metadata.NewDeviceServiceClient(params, consulEndpoint)
 
 	params.Path = v1Deviceprofile
 	params.Url = metaAddr + params.Path
-	svc.dpc = metadata.NewDeviceProfileClient(params, types.Endpoint{})
+	svc.dpc = metadata.NewDeviceProfileClient(params, consulEndpoint)
 
 	params.Path = v1Schedule
 	params.Url = metaAddr + params.Path
-	svc.scc = metadata.NewScheduleClient(params, types.Endpoint{})
+	svc.scc = metadata.NewScheduleClient(params, consulEndpoint)
 
 	params.Path = v1ScheduleEvent
 	params.Url = metaAddr + params.Path
-	svc.scec = metadata.NewScheduleEventClient(params, types.Endpoint{})
+	svc.scec = metadata.NewScheduleEventClient(params, consulEndpoint)
 
 	// initialize Core Data clients
 	params.ServiceKey = svc.c.Clients[ClientData].Name
 
 	params.Path = v1Event
 	params.Url = dataAddr + params.Path
-	svc.ec = coredata.NewEventClient(params, types.Endpoint{})
+	svc.ec = coredata.NewEventClient(params, consulEndpoint)
 
 	params.Path = v1Valuedescriptor
 	params.Url = dataAddr + params.Path
-	svc.vdc = coredata.NewValueDescriptorClient(params, types.Endpoint{})
+	svc.vdc = coredata.NewValueDescriptorClient(params, consulEndpoint)
+
+}
+
+func checkClientsInitialSuccessful() {
```

Comment:
this one should checks successful initialization of all clients or only schedules and scheduleEvents? From function name description - all, but in code I see only two. Maybe better add comments for function name or change it accordingly?

Reply:
ok, I will add comments on it.